JenkinsfileRT_dev

```diff
@@ -1,24 +1,14 @@
 // [skip ci] and [ci skip] have no effect here.
 if (utils.scm_checkout(['skip_disable':true])) return
 
-def PipInject(String reqs) {
```

Comment:
This is a feature that I regularly use.  I don't know if it's worth keeping around just for me.

Reply:
Ah, you do use it?!  Ok, then we'll keep it in.

Reply:
This is typically how I test an asdf branch, just add a `git+` requirement to the input box on the parametrized build.  Saves me from having to create a jwst branch with an updated requirements-dev.txt

Reply:
I will add that capability to the normal nightly build too then.